src/pretix/base/migrations/0239_auto_20230510_1017.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.18 on 2023-05-10 10:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pretixbase', '0238_giftcard_owner_ticket'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='itemmetaproperty',
+            name='allowed_values',
+            field=models.TextField(null=True),
```

Comment:
See below, as this is auto-generated, probably do not update by hand. But for the sake of completeness.
```suggestion
            field=models.TextField(),
```

Reply:
As discussed, it will continue to be nullable :)